### PR TITLE
RNG

### PR DESCRIPTION
**Pull request by jtormey**
Ports rng module from v3, mixes local and server entropy.

### CODE REVIEW

File: packages/blockchain-wallet-v4/src/walletCrypto/rng.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+// getServerEntropy :: Int -> Promise (Either Buffer Error) Error
+export const getServerEntropy = (nBytes = DEFAULT_BYTES, api) => {
+  if (!isPositiveInteger(nBytes)) {
+    return Promise.resolve(Either.Left(
+      new Error('Must provide a positive integer to getServerEntropy')
+    ))
+  }
+
+  return api.getRandomBytes(nBytes, FORMAT).then(responseText => {
```
Comment (mpfluger) [inline packages/blockchain-wallet-v4/src/walletCrypto/rng.js]: Are non 200 error codes handled further upstream and result in an error that's caught and re-thrown in the catch below?

**Comment by mpfluger** [comment]
@akly and @kristovatlas please review this as well. Most of the code is from the V3 wallet with some minor refactoring: https://github.com/blockchain/My-Wallet-V3/blob/master/src/rng.js.

File: packages/blockchain-wallet-v4/src/walletCrypto/rng.js (hunk 1 above)
Comment (jtormey) [reply packages/blockchain-wallet-v4/src/walletCrypto/rng.js]: A failed request will cause the promise returned by `createRng` to reject, so it's up to the caller to handle it. The code in this block only runs if the call randombytes was successful.

Comment (mpfluger) [reply packages/blockchain-wallet-v4/src/walletCrypto/rng.js]: OK, thanks!

File: packages/blockchain-wallet-v4/package.json [hunk 2]
```diff
@@ -34,6 +34,15 @@
       "<rootDir>/lib"
     ]
   },
+  "standard": {
```
Comment (schnogz) [inline packages/blockchain-wallet-v4/package.json]: Shouldn't need this since we use ESLint and utilize Jest env support in the root `.eslintrc` file.

Comment (schnogz) [reply packages/blockchain-wallet-v4/package.json]: Also PR #263 adds support for ECMAScript 8 which once merged will fix your lint "violations"

Comment (jtormey) [reply packages/blockchain-wallet-v4/package.json]: Thanks! Will fix

File: packages/blockchain-wallet-v4/src/utils/checks.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { is } from 'ramda'
+
+/* Number */
+export const isNumber = is(Number)
```
Comment (pernas) [inline packages/blockchain-wallet-v4/src/utils/checks.js]: for the payment branch we have those things defined in the payment file. We will need to remember to use these ones too :D

Comment (jtormey) [reply packages/blockchain-wallet-v4/src/utils/checks.js]: Yes that's a good idea

**Pull request merged**